go.mod

```diff
@@ -54,6 +54,8 @@ require (
 	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c // indirect
 	golang.org/x/term v0.0.0-20210503060354-a79de5458b56 // indirect
 	golang.org/x/text v0.3.6 // indirect
+	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c // indirect
```

Comment:
can we run `go mod tidy`? I don't think the main copilot binary needs a dependency on `grpc` 🙏

Reply:
Sorry. I will fix it. 🙏